### DISCUSSION TITLE
what does "Need admin approval" mean?

### DISCUSSION THREAD

**Discussion by ryao-mdanderson** [abraunegg/onedrive#1924] [General]
#Greetings,

We set up onedrive on a linux system inside the organization, when launched 'onedrive' and provided the institution email address and password,  then,  received the window say:

 Need admin approval
    ...
 
    needs permission to access resources in your organization
    that only an admin can grant. Please ask an admin to
    grant permission to this app before you can use it.

May anyone explain and provide a suggestion how to proceed? Thanks!

**Comment by abraunegg** [comment]
@ryao-mdanderson 
What that means is your domain admin needs to approve - please speak to your domain admins

**Comment by ryao-mdanderson** [comment]
@abraunegg : Thank you! domain admins, I think you refer to the administrator for Onedrive in our organization.
